fontbakery-check-ttf.py

```diff
@@ -1724,30 +1725,16 @@ def check_bit_entry(font, table, attr, expected, bitmask, bitname):
 
     # ----------------------------------------------------
     fb.new_check("Checking with ftxvalidator")
-    KNOWN_GOOD_OUTPUT = ''\
```

Comment:
AFAIK, this kind of output won't be generated with the `-r` flag ("full report") also, the order of the items in the `kATSFontTestResultKey` array can be different than here.